testplan/testing/multitest/driver/base.py

```diff
@@ -464,15 +432,49 @@ def get_lines_at_tail(log_file, max_count):
 
         return content
 
+    @property
+    def identifier(self):
+        return None
+
+    @property
+    def local_port(self):
+        return None
+
+    @property
+    def local_host(self):
```

Comment:
can we just call these host and port?
maybe don't need this dummy property in base class.
only add these properties to drivers who uses a BaseConnectionExtractor?